src/extensions/default/Inline3DParametersEditor/css/main.less

```diff
@@ -0,0 +1,29 @@
+div.vec3 {
```

Comment:
If it's possible to do everything in a `.css` vs. needing to use a `.less` file, it would be good, since it reduces us having to process the file.